### ISSUE TITLE
Most Mods have a MiniMap

### ISSUE THREAD

**Issue by Badcholo** [simeonradivoev/MatterOverdrive#245]
Most mods have a mini map of some sort. When you turn into an android the abilities show up under the minimap and are blocked by it. It would be nice to have a way to move them or swap them all to the left side. options on this would be nice. Smaller, to the left and not so transparent.

**Comment by simeonradivoev** [comment]
You can. In the Android Station in the config page there is a button that opens options for the Android HUD. You can change colors, positions and opacity.

**Comment by Badcholo** [comment]
Thanks
